code/modules/mob/living/carbon/xenomorph/castes/ravager/abilities_ravager.dm

```diff
@@ -122,6 +122,9 @@
 	atoms_to_ravage += get_step(owner, turn(owner.dir, -45)).contents
 	atoms_to_ravage += get_step(owner, turn(owner.dir, 45)).contents
 	for(var/atom/movable/ravaged AS in atoms_to_ravage)
+		if(ishitbox(ravaged) || isarmoredvehicle(ravaged))
```

Comment:
why only armoured vehicles? Why not just all vehicles

Reply:
Couldn't this also go after the !ishuman actually

Reply:
It cannot go after !ishuman. Doing so causes it to just not work unless it hits the center of the vehicle. I can change it to isvehicle though.

Reply:
The same applies with trying to add the vehicle/hitbox checks into the !ishuman check. Not really sure why !ishuman doesn't actually let the ravage normally attack the hitboxes.

Reply:
Ok not a biggie